keyboards/keybee/keybee65/readme.md

```diff
@@ -0,0 +1,21 @@
+# KeyBee 65
+
+The following is the QMK Firmware for the KeyBee 65 Hotswap PCB.
+
+The PCB features:
+* QMK & VIA compatibility
+* Hotswap
+* RGB underglow
+* Per-Key RGB
+
+---
+
+* Keyboard Maintainer: [ToastyStoemp](https://github.com/ToastyStoemp)
+* Hardware Supported: Pollen65
+* Hardware Availability: [keybeeshop.com](https://www.keybeeshop.com/collections/keyboards/products/pollen65-mechanical-keyboard-kit)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Reply:
Done